surveys/2021-annual-survey/questions.md

```diff
@@ -201,9 +198,11 @@ Free form.
 
 ### How would you rate your Rust expertise?
 
-Select 1-10 (1 = Beginner, 10 = Expert)
+Select 0-4 (0 = I can't write or read Rust, 1 = I can write simple exercises in Rust, 2 = I can write useful, production-ready code but it is a struggle, 3 = I am productive writing Rust, 4 = I'm an expert)
```

Comment:
Nit: 0 based indexing in a survey is cute but a bit unusual. 

I would also format this by putting each option in a different line.